index.html

```diff
@@ -242,12 +244,37 @@ <h2 class="language">Skills</h2>
 
 
             </section>
-            
+
+
         </section>
     </div>
+    <section class="white">
+    <footer class="footer white">
+        <div class="foot">
+        <h2 class="contact-text">Contact me</h2>
+        <p class="contact-paragraph">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+        </p>
+        <form>
+            <label for="fname">Full name:</label>
+            <input id="fname" type="text" maxlength="30" name="full-name" placeholder="Yeremias" required class="input">
+
+            <label for="email">Email</label>
+            <input id="email" type="email" name="email-address" placeholder="natajayanj14@gmail.com" required class="input">
+
+            <label for="textarea">Message:</label>
+            <textarea id="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="Write your message here" required class="input text-area"></textarea>
+            <button class="btn" type="submit">Get in touch</button>
+        </form>
```

Comment:
- [ ] The form should have an `action` and method `POST` like the requirement says. see screenshot 👇
![rr](https://user-images.githubusercontent.com/61562626/129794183-2b8a5576-d88e-428d-a7ef-7b7c08f8ee9f.jpeg)